### PR TITLE
Package coq-cfml-stdlib.20211215

### PR DESCRIPTION
**Pull request by fpottier**
### `coq-cfml-stdlib.20211215`
The CFML program verification system
CFML is a program verification system for OCaml.

---
* Homepage: https://gitlab.inria.fr/charguer/cfml2
* Source repo: git+https://gitlab.inria.fr/charguer/cfml2.git
* Bug tracker: https://gitlab.inria.fr/charguer/cfml2/-/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

**Comment by palmskog** [comment]
@fpottier since this actually depends on the `coq-cfml-basis` package you submitted in another PR, this is going to take a while to get merged (I have to first get that package merged and then restart CI here).

File: released/packages/coq-cfml-stdlib/coq-cfml-stdlib.20211215/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+synopsis: "The CFML program verification system"
+description: """
+CFML is a program verification system for OCaml.
+"""
+
+build: [make "-C" "lib/stdlib" "-j%{jobs}%"]
+install: [make "-C" "lib/stdlib" "install"]
+depends: [
+  "coq" { >= "8.13" }
+  "cfml" { = version }
```
Comment (palmskog) [inline released/packages/coq-cfml-stdlib/coq-cfml-stdlib.20211215/opam:19]: Where is this package defined? Is it really upstream in OCaml?

Comment (fpottier) [reply released/packages/coq-cfml-stdlib/coq-cfml-stdlib.20211215/opam:19]: Hi Karl, the `cfml` package should appear in the OCaml package repository soon, I have made a PR for it. Perhaps I should have waited for it to appear before submitting this PR, sorry.

Comment (fpottier) [reply released/packages/coq-cfml-stdlib/coq-cfml-stdlib.20211215/opam:19]: Also, I should have submitted `coq-cfml-stdlib` and `coq-cfml-basis` in the same PR, sorry. (I will do so next time.)

Comment (palmskog) [reply released/packages/coq-cfml-stdlib/coq-cfml-stdlib.20211215/opam:19]: Sounds good, maybe you can do a force-push to to the PR once the `cfml` package has been merged. In the meantime, we can at least merge `coq-cfml-basis`.

**Comment by fpottier** [comment]
The package `cfml` has been accepted in opam on the OCaml side.

**Comment by palmskog** [comment]
@fpottier unfortunately it can take up to day from a package being merged in the OCaml opam repository to the package actually being available when one does `opam update`. Since we would like CI to pass here, I will wait until tomorrow and then rerun CI. However, if you monitor locally you can always do a force-push if you see that `cfml` appears after `opam update`.

**Pull request merged**